python/PEDvPUDplot.py

```diff
@@ -2,104 +2,166 @@
 """
 Created on Sat Nov  7 12:59:47 2015
 
-@author: ymseah
+Make scatterplot of pairwise uncorrected vs evolutionary distances.
+
+Pairwise uncorrected distances are obtained from SDT 1.0 (filename_mat.txt), while pairwise maximum likelihood distances are obtained from Tree-Puzzle 5.2 (outdist.txt). Both files are read to extract sequence ID and distance values.
+
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
-import itertools # for zipping, less memory consumption than zip
+import itertools
 
 filepath = raw_input("Enter path to SDT matrix file: ")
 filepath2 = raw_input("Enter path to TreePuzzle distance output file: ")
 
-# function to get sequence IDs from SDT matrix output file
 def getSDTid(filepath):
+    """
+    Reads SDT 1.0 matrix output (filename_mat.txt) and returns list of sequence IDs.
+    
+    ::param filepath::  Path to SDT file (input).
+    ::param sdtfile::   Input file.
+    ::param seqid::     List of sequence ID strings.
+    ::param eachseqid:: Individual sequence ID string.
+    ::param line::      Line from input file.
+    ::param eachline::  List of elements from param line.
+    """
     with open(filepath) as sdtfile:
         seqid = []
-        for lines in sdtfile:
-            eachline = lines.split(',')
-            seqid.append(eachline[0])
+        eachseqid = ""
+        for line in sdtfile:
+            eachline = line.split(',')
+            eachseqid = eachline[0]
+            if len(eachseqid) > 10:
+                eachseqid = eachseqid[:10]
+                seqid.append(eachseqid)
+            else:
+                seqid.append(eachseqid)
     return seqid
 
-# function to calculate distance from similarity, and vice-versa (complement)
-def comp(num):
-    compnum = 1-num
-    return compnum
-
-# function to list all SDT distances
 def getSDTdist(filepath):
+    """
+    Reads SDT 1.0 matrix output (filename_mat.txt) and returns list of pairwise distances as float numbers. 
```

Comment:
Moved this from a commit to here:
Don't mention filename_mat.txt as it's too specific. In general, you try to generalize a function as much as possible so specific names like this are not helpful unless the data follows a specific structure then you can say, "See for data structure." Then provide this file or a sample file in your repository so people can look at it.